src/libwalletqt/Wallet.h

```diff
@@ -348,6 +349,10 @@ class Wallet : public QObject
     Q_INVOKABLE void segregationHeight(quint64 height);
     Q_INVOKABLE void keyReuseMitigation2(bool mitigation);
 
+    // Passphrase entry for hardware wallets
+    Q_INVOKABLE void onPassphraseEntered(const QString &passphrase, bool enter_on_device, bool entry_abort=false);
+    Q_INVOKABLE virtual void onWalletPassphraseNeeded(bool on_device) override;
```

Comment:
Do you plan to invoke this from QML?

Reply:
Currently, `onPassphraseEntered` is called by the QML, `onWalletPassphraseNeeded` is not (is called by passphrase helper from the wallet listener, causes signal emission to the QML, displays prompt). Should I remove `Q_INVOKABLE` from `onWalletPassphraseNeeded` then?

Reply:
Yep, would prefer removing `Q_INVOKABLE ` from `onWalletPassphraseNeeded` if we don't call it from QML.

Reply:
Done, pls check

Reply:
Looks okay to me.
Only the PR author can `Resolve` it.